### PR TITLE
Create ui-c8y-1021-0-0-typeahead-not-updated-on-click-selection.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by dafkk

Ticket: [DM-3877](https://cumulocity.atlassian.net/browse/DM-3877)
Original PR: [7245](https://github.com/Cumulocity-IoT/cumulocity-ui/pull/7245)

[DM-3877]: https://cumulocity.atlassian.net/browse/DM-3877?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/change-logs/device-management/ui-c8y-1021-0-0-typeahead-not-updated-on-click-selection.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Typeahead not updated on click selection.
```
Comment (gen-App-Enablement) [inline content/change-logs/device-management/ui-c8y-1021-0-0-typeahead-not-updated-on-click-selection.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Typeahead field now updates correctly when selecting a value by mouse click
```

File: content/change-logs/device-management/ui-c8y-1021-0-0-typeahead-not-updated-on-click-selection.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component--KIsStyzM
+    label: Device Management app
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: DM-3877
+version: 1021.0.0
+---
+Typeahead not updated on click selection.
```
Comment (gen-App-Enablement) [inline content/change-logs/device-management/ui-c8y-1021-0-0-typeahead-not-updated-on-click-selection.md:17]: The Claude AI suggested to use the following body:
```suggestion
Previously, when a user selected a value from the suggestions in a typeahead field by clicking it with the mouse, the field was not properly updated with the selected value. This has now been fixed. With this change, the typeahead field correctly shows the selected value when the user clicks on a suggestion in the dropdown list. This improves the usability and consistency of typeahead fields across the application.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Comment by reey** [comment]
Closing as already covered in: https://github.com/Cumulocity-IoT/c8y-docs/pull/2530

**Pull request closed without merging**